buildSrc/build.gradle

```
@@ -6,10 +6,12 @@ plugins {
 // So we'll load it into our own object to grab what we need.
 def constants = new Properties()
```

```suggestion
Properties constants = new Properties()
```

just for consistency